sdk/eventhub/Azure.Messaging.EventHubs/tests/EventSenders/EventSenderLiveTests.cs

```diff
@@ -154,8 +154,8 @@ await using (var sender = client.CreateSender())
         }
 
         /// <summary>
-        ///   Verifies that the <see cref="EventHubClient" /> is able to
-        ///   connect to the Event Hubs service.
+        ///   Verifies that the <see cref="EventSender" /> is able to
+        ///   connect to the Event Hubs service. and perform operations.
```

Comment:
"service. and perform" -> "service and perform"

Reply:
There are a few other instances up there. :D

Reply:
Typing is hard...